docs/guides/setup/configuring_data_contexts/how_to_instantiate_a_data_context_without_a_yml_file.md

```diff
@@ -19,176 +19,222 @@ Steps
-                    "bucket":  "my_data_docs_bucket",
-                    "prefix":  "my_optional_data_docs_prefix",
+        "evaluation_parameter_store": {"class_name": "EvaluationParameterStore"},
+    },
+    expectations_store_name="expectations_S3_store",
+    validations_store_name="validations_S3_store",
+    evaluation_parameter_store_name="evaluation_parameter_store",
+    data_docs_sites={
+        "s3_site": {
+            "class_name": "SiteBuilder",
+            "store_backend": {
+                "class_name": "TupleS3StoreBackend",
+                "bucket":  "my_data_docs_bucket",
+                "prefix":  "my_optional_data_docs_prefix",
+            },
+            "site_index_builder": {
+                "class_name": "DefaultSiteIndexBuilder",
+                "show_cta_footer": True,
+            },
+        }
+    },
+    validation_operators={
+        "action_list_operator": {
+            "class_name": "ActionListValidationOperator",
+            "action_list": [
+                {
+                    "name": "store_validation_result",
+                    "action": {"class_name": "StoreValidationResultAction"},
                 },
-                "site_index_builder": {
-                    "class_name": "DefaultSiteIndexBuilder",
-                    "show_cta_footer": True,
+                {
+                    "name": "store_evaluation_params",
+                    "action": {"class_name": "StoreEvaluationParametersAction"},
                 },
-            }
-        },
-        validation_operators={
-            "action_list_operator": {
-                "class_name": "ActionListValidationOperator",
-                "action_list": [
-                    {
-                        "name": "store_validation_result",
-                        "action": {"class_name": "StoreValidationResultAction"},
-                    },
-                    {
-                        "name": "store_evaluation_params",
-                        "action": {"class_name": "StoreEvaluationParametersAction"},
-                    },
-                    {
-                        "name": "update_data_docs",
-                        "action": {"class_name": "UpdateDataDocsAction"},
-                    },
-                ],
-            }
-        },
-        anonymous_usage_statistics={
-          "enabled": True
+                {
+                    "name": "update_data_docs",
+                    "action": {"class_name": "UpdateDataDocsAction"},
+                },
+            ],
         }
-    )
-    ```
+    },
+    anonymous_usage_statistics={
+      "enabled": True
+    }
+)
+```
 
 
-2. **Pass this DataContextConfig as a project_config to BaseDataContext**
+### 2. Pass this DataContextConfig as a project_config to BaseDataContext
 
-    ```python
-    context = BaseDataContext(project_config=data_context_config)
-    ```
+```python
+from great_expectations.data_context.types.base import BaseDataContext
```

Comment:
Added this so they can just copy paste the last bit

Reply:
Thank you!